package.json

```diff
@@ -17,16 +17,17 @@
     "i18n:compile": "yarn i18n:extract && lingui compile",
     "i18n:pseudo": "lingui extract --locale pseudo && lingui compile",
     "prepare": "yarn contracts:compile && yarn graphql:fetch && yarn graphql:generate && yarn i18n:compile",
+    "postinstall": "patch-package",
```

Comment:
I reordered these to match the app lifecycle, and only felt comfortable doing so because I was already touching this file. I don't think it requires a separate PR.